src/web/mjs/connectors/NovelRingan.mjs

```diff
@@ -0,0 +1,88 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class NovelRingan extends Connector {
+    constructor() {
+        super();
+        super.id = 'novelringan';
+        super.label = 'Novel Ringan';
+        this.tags = ['novel', 'indonesian'];
+        this.url = 'https://novelringan.com';
+
+        this.novelContainerQuery = 'main.content';
+        this.novelContentQuery = 'div.entry-content';
+        this.novelFormat = 'image/png';
+        this.novelWidth = '56em';
+        this.novelPadding = '1.5em';
+    }
+
+    async _getMangas() {
+        let request = new Request(
+            this.url + '/daftar-novel',
+            this.requestOptions
+        );
+        let data = await this.fetchDOM(request, 'div.blix ul li a');
+        return data.map((element) => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim(),
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(this.url + manga.id, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.bxcl ul li a');
+        return data.map((element) => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim(),
+                language: '',
+            };
+        });
+    }
+
+    async _getPagesNovel(request) { //Adapted from Novelgo.mjs
```

Comment:
Content of this method may be merged into `_getPages()`, since there seems to be no further benefit in splitting this up (it is always used with the request provided by `_getPages()`)